include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -149,6 +149,18 @@ struct gap
     //!\}
 };
 
-constexpr gap gap::GAP{};
+/*!\name Literals
+ * \{
+ */
+/*!\brief The seqan3::gap char literal.
+ * \param[in] c doesn't really matter, as gap only has one character.
```

Comment:
Does this mean you can assign any character? :thinking:

Reply:
Hm alright, but then I would change this, as gap does not "have" a character.
```suggestion
 * \param[in] c Placeholder; A gap object will be created regardless of the given character.
```

Reply:
oh and add a test for this! assigning another char with he literal like `'f'_gap`. Although this is super wierd...